funsite/locale/fr/LC_MESSAGES/django.po

```diff
@@ -135,6 +135,10 @@ msgstr "Erreur serveur"
 msgid "Sign up"
 msgstr "Inscription"
 
+#: /edx/app/edxapp/fun-apps/funsite/templates/lms/login.html:37
+msgid "Sign up for FUN today!"
+msgstr "Inscrivez-vous sur FUN !"
```

Comment:
"today" ? Si on veut garder l'idée d'instantanéité, on peu parler de "now", mais dans ce cas faut traduire en français, non ?